OSLC4Net_SDK/scripts/vocab_gen.py

```
@@ -0,0 +1,157 @@
+import rdflib
+import argparse  # Import the argparse library
+import sys       # To exit gracefully on error
+from rdflib import Graph, URIRef, Namespace
```

Ruff (0.8.2)

`rdflib.Namespace` imported but unused

Remove unused import: `rdflib.Namespace`

(F401)